openssl-compat.h

```diff
@@ -1,47 +1,9 @@
 #ifndef OPENSSL_COMPAT_H
 #define OPENSSL_COMPAT_H
 
-#include <openssl/bio.h>
 #include "util-internal.h"
+#include <openssl/bio.h>
 
-#if (OPENSSL_VERSION_NUMBER < 0x10100000L) || \
-	(defined(LIBRESSL_VERSION_NUMBER) && LIBRESSL_VERSION_NUMBER < 0x20700000L)
```

Comment:
This is `false` in our case, and also the condition seems correct, please revert.